extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -564,7 +571,7 @@ describes.realWin('amp-ad-3p-impl', {
           // Right margin is 9px from containerContainer and 25px from
           // container.
           expect(element.style.marginLeft).to.be.equal('');
-          expect(element.style.marginRight).to.be.equal('-49px');
+          expect(element.style.marginRight).to.be.equal('-34px');
```

Comment:
was the testing continuously failing?

Reply:
hmmmm. According to the comment we should put -34px. 
The test is very closely related to the browser size. For some reason, it keeps failing when I was debugging yesterday. But somehow it works today....... Anyhow I reverted the change.